arm-containerregistry/2016-06-27-preview/swagger/containerregistry.json

```diff
@@ -323,6 +323,46 @@
         },
         "deprecated": false
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.ContainerRegistry/registries/{registryName}/regenerateCredentials": {

```

Comment:
instead of resourceGroup the name of the path parameter should be resourceGroupName. This will be consistent with other specs.